scripts/run_performance_tests.sh

```diff
@@ -68,7 +68,9 @@ fi
 trap cleanup EXIT
 
 # Change execute status of browsermob-proxy
-chmod +x $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
+# export ME=$(whoami)
+# chown $ME $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
```

Comment:
Did you intend to remove these two comments? Please do. We don't want to introduce cruft.